business/MetadataDatabase/Configuration/AuthConfiguration.cs

```diff
@@ -0,0 +1,8 @@
+namespace MetadataDatabase.Models
+{
+    public class AuthConfiguration
+    {
+        public string Secret { get; set; }
```

Comment:
What is 'Secret' ?

Reply:
The secret key is combined with the header and the payload to create a unique hash. You are only able to verify this hash if you have the secret key.

For more information : https://stackoverflow.com/questions/31309759/what-is-secret-key-for-jwt-based-authentication-and-how-to-generate-it/31313582